### DISCUSSION TITLE
Trying to configure linkerd-cli after deploying linkerd with helm

### DISCUSSION THREAD

**Discussion by parsaloi** [linkerd/linkerd2#12746] [Help]
After successfully deploying the linkerd-edge/linkerd-control-plane with helm I have tried to inspect the deployment with the linkerd-cli. I get the following on the command line:
```
$ linkerd check
kubernetes-api
--------------
√ can initialize the client
√ can query the Kubernetes API

kubernetes-version
------------------
√ is running the minimum Kubernetes API version

linkerd-existence
-----------------
× 'linkerd-config' config map exists
    configmaps "linkerd-config" not found
    see https://linkerd.io/2/checks/#l5d-existence-linkerd-config for hints

Status check results are ×
```

I looked at the troubleshooting notes and verified the config is present in the cluster:
```
kubectl -n linkerd get configmap
NAME                           DATA   AGE
kube-root-ca.crt               1      49m
linkerd-config                 2      18m
linkerd-identity-trust-roots   1      18m

```

**Comment by alpeb** [comment]
That's really strange... are the linkerd command and kubectl using the same kubeconfig in your environment?

**Reply by parsaloi** [reply]
I have been trying to figure out how to configure the linkerd-cli to use the kubeconfig that kubectl is using which is yet to work...
